### PR TITLE
PIM-4745 Add new style for the notifications

### PR DESCRIPTION
**Pull request by yulldesign**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
         <span class="caret"></span>
     </button>
     <ul class="dropdown-menu right-align">
+        <li></li>
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: not possible to add it with css ? For me it's not an acceptable solution

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 2]
```diff
@@ -29,11 +29,16 @@ define(
             template: _.template(
                 [
                     '<a href="<%= url ? url : \'javascript: void(0);\' %>"<%= viewed ? \'\' : \'class="new"\' %>>',
-                        '<i><%= actionTypeMessage %></i>',
-                        '<i class="icon-<%= icon %>"></i>',
-                        '<%= message %>',
-                        '<i><%= createdAt %></i>',
+                        '<div class="action_icon <%= actionType %>">',
+                          '<i class="status icon-<%= icon %>"></i>',
```
Comment (sumbobyboys) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: please use 4 space indentation

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 3]
```diff
@@ -29,11 +29,16 @@ define(
...
+                        '</div>',
+                        '<div class="metas">',
+                          '<span class="title"><%= actionTypeMessage %></span>',
```
Comment (sumbobyboys) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: same here

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 4]
```diff
@@ -20,8 +20,8 @@
 
   .filter-item {
     .filter-criteria-selector, [class$=dropdown-toggle].btn {
-      height:29px;
-      line-height: 29px;
+      height:30px;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: please add a space

File: src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less [hunk 5]
```diff
@@ -0,0 +1,192 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+#oroplatform-header .nav #header-notification-widget {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less]: as there is only one #header-notification-widget, you don't need `#oroplatform-header .nav`

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 6]
```diff
@@ -29,11 +29,16 @@ define(
...
+                          '<span class="message"><%= message %></span>',
+                        '</div>',
+                        '<time><%= createdAt %></time>',
                         '<i class="icon-<%= viewed ? \'trash\' : \'eye-close\' %> action"></i>',
+                        '<button class="btn icons-holder-text"><i class="icon-file-text-alt"></i>Report</button>',
```
Comment (phaseinducer) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: 12 lines is long enough to make a separate html file imho, way more clean and maintainable

**Comment by phaseinducer** [comment]
4 spaces indentation everywhere if possible

File: src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less (hunk 5 above)
Comment (yulldesign) [inline src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less]: i need it, its the same in pim.less today

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js (hunk 6 above)
Comment (yulldesign) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: if you could help me for that, its beyond my competence.

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html (hunk 1 above)
Comment (yulldesign) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: i don't have another solution

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: yep, you just didn't found it :)

File: src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less (hunk 5 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less]: so there is a problem: an id should be unique. If it's not, it should be converted to a class

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html (hunk 1 above)
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: not a big deal for me, i prefer see this fixed and master

File: src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less (hunk 5 above)
Comment (yulldesign) [inline src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less]: ID is unique

Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/notifications.less]: ok, so drop `#oroplatform-header .nav` part

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html (hunk 1 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: I'm not ok with that, but well..

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/save-buttons.html]: I found the solution, you can remove this crap :)

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 7]
```diff
@@ -29,11 +29,16 @@ define(
             template: _.template(
                 [
                     '<a href="<%= url ? url : \'javascript: void(0);\' %>"<%= viewed ? \'\' : \'class="new"\' %>>',
```
Comment (sumbobyboys) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: Could we use a template here ?

Comment (grena) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: Could be nice indeed

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js [hunk 8]
```diff
@@ -29,11 +29,16 @@ define(
...
-                        '<i><%= actionTypeMessage %></i>',
-                        '<i class="icon-<%= icon %>"></i>',
-                        '<%= message %>',
-                        '<i><%= createdAt %></i>',
+                        '<div class="action-icon <%= actionType %>">',
```
Comment (grena) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: to replace underscores by '-', please use 

```
<%= actionType.replace("_", "-") %>
```

File: src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js (hunk 7 above)
Comment (nidup) [inline src/Pim/Bundle/NotificationBundle/Resources/public/js/notification-list.js]: i need to rework once merged i can do that

**Comment by grena** [comment]
Replaced by https://github.com/akeneo/pim-community-dev/pull/3104

**Pull request closed without merging**